### DISCUSSION TITLE
autoBuild onFileChange does not work

### DISCUSSION THREAD

**Discussion by lars-soem** [James-Yu/LaTeX-Workshop#3116] [Q&A]
Please see [this stackoverflow question](https://stackoverflow.com/questions/54824914/vscode-latex-workshop-no-autobuild-for-onfilechange) for further details.

I experience the same problems with LaTeX Workshop 8.16.1 in vscode 1.54.3

**Comment by tamuratak** [comment]
- Fill out all the information required in [the issue template](https://github.com/James-Yu/LaTeX-Workshop/blob/master/.github/ISSUE_TEMPLATE/bug-report.md). 
- Please paste the whole log messages here, not parts of ones. It should start with `Initializing LaTeX Workshop`. It is very important to identify problems.

![スクリーンショット 2020-02-14 20 41 25](https://user-images.githubusercontent.com/10665499/74528024-87b2ef80-4f6a-11ea-9c5d-4f9feff25524.png)

When all of the above is done, we will reopen this issue if needed.
